test-app/tests/integration/components/bs-form/element-test.gjs

```diff
@@ -1312,17 +1501,26 @@ module('Integration | Component | bs-form/element', function (hooks) {
   testRequiringFocus(
     'events enabling validation rendering are configurable per `showValidationOn` (array)',
     async function (assert) {
-      this.set('errors', A(['Invalid']));
-      this.set('model', EmberObject.create({ name: null }));
-      this.set('showValidationOn', ['change']);
-      this.set('formElement', ensureSafeComponent(ValidatingFormElement, this));
-
-      await render(hbs`<this.formElement
-  @property='name'
-  @errors={{this.errors}}
-  @model={{this.model}}
-  @showValidationOn={{this.showValidationOn}}
-/>`);
+      class State {
+        @tracked errors = A(['Invalid']);
```

Comment:
`errors` does not seem to be changed after initial rendering in this case. Or did I missed something?